scripts/build_imagemagick

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+set -e
+
+WORK_DIR=/usr/src
+
+function handle_error() {
+  local status=$?
+  if [[ $status > 0 ]]; then
+    echo "🔥 There was a problem building the binary. Operation aborted."
+    exit 1
+  fi
+}
+trap handle_error EXIT SIGINT
```

Comment:
Here we detect Signal Interruption (aka Ctrl+C in the terminal) or any `EXIT` statement from any command. Whenever that happens and the status `is > 0` we show an error message.

Reply:
I really like how in control we are of how the errors are handled here!